IBPSA/BoundaryConditions/WeatherData/BaseClasses/getTimeSpanWeatherData.mo

```diff
@@ -0,0 +1,87 @@
+within IBPSA.BoundaryConditions.WeatherData.BaseClasses;
+function getTimeSpanWeatherData
+  "Get the time span of the weather data from the file"
+
+  input String filNam "Name of weather data file";
+  input String tabName "Name of weather table";
+  output Modelica.SIunits.Time[2] timeSpan "start and end time of weather data";
```

Comment:
Please start comments with capital letters (also below)

Reply:
Good to know. Hope I did it everywhere I added comments..